.github/workflows/mobsf.yml

```
@@ -33,7 +33,7 @@ jobs:
           python-version: 3.8
 
       - name: Run mobsfscan
-        uses: MobSF/mobsfscan@d564d5858face141caf984437c922a9bc55b03b0
+        uses: MobSF/main
```

actionlint (1.7.4)

specifying action "MobSF/main" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)